UsabillaSampleApp/android/build.gradle

```diff
@@ -3,7 +3,7 @@
 buildscript {
     ext {
         buildToolsVersion = "29.0.2"
-        minSdkVersion = 19
+        minSdkVersion = 21
```

Comment:
Why do we need to upgrade this?

Reply:
When I try to run the sample app, there is an error saying your must target sdk 21 min.